tool/tsh/common/tsh.go

```diff
@@ -2086,14 +2086,36 @@ func onLogout(cf *CLIConf) error {
 		proxyHost = cf.Proxy
 	}
 
-	switch {
-	// Proxy and username for key to remove.
-	case proxyHost != "" && cf.Username != "":
-		tc, err := makeClient(cf)
+	tc, err := makeClient(cf)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	var SAMLSingleLogoutURL string
+	clt, err := tc.ConnectToCluster(cf.Context)
```

Comment:
How is this working with multiple profiles if we're only ever connecting to a single cluster once?